client/lib/resource/src/scope-form/scope-form.hbs

```diff
@@ -78,6 +88,10 @@
                             id="scope_appScope"
                             value="scope_appScope"
                             name="yourturn_scope_scopeType"
+                            {{#if scope.ownerScope}}
```

Comment:
Instead of `{{#if condition}}{{else}}thing{{/if}}` you could just do `{{#unless condition}}thing{{/unless}}`.